install_config/registry/index.adoc

```diff
@@ -37,11 +37,3 @@ installed separately as a stand-alone container image registry.
 To install a stand-alone registry, follow
 xref:../../install/stand_alone_registry.adoc#install-config-installing-stand-alone-registry[Installing a Stand-alone Registry]. This installation path deploys an all-in-one cluster
 running a registry and specialized web console.
-
```

Comment:
@chrisnegus Since this topic is for the internal registry, I moved the TBR (auth enabled) and Quay content out. I moved the Quay content to the Architecture guide:  architecture/infrastructure_components/image_registry.adoc